blocks/header/header.css

```diff
@@ -1,12 +1,22 @@
 /* header and nav layout */
-header .nav-wrapper {
+header {
+  border-bottom: 2px solid var(--col-neutral20);
+}
+
+.nav-wrapper {
+  --nav-wrapper-padding-block: 26px;
+
+  padding-block: var(--nav-wrapper-padding-block);
   background-color: var(--background-color);
   width: 100%;
-  z-index: 2;
-  position: fixed;
+
+  @media (min-width: 768px) {
+    --nav-wrapper-padding-block: 10px;
+  }
 }
 
-header nav {
+nav {
```

Comment:
to avoid issues in case we include another nav, can you use classname or id "#nav"?